src/sentry/static/sentry/app/components/forms/selectAsyncField.jsx

```diff
@@ -65,7 +65,9 @@ class SelectAsyncField extends SelectField {
   // Otherwise, when you hit "enter" to create a new item, the "selected value" does
   // not update with new value (and also new value is not displayed in dropdown)
   coerceValue(value) {
```

Comment:
theres an open question if coerceValue and onChange logic needs duplicated.. probably not and I could at least combine them into coerceValue